<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<title>[Jack-Devel] Restoring trac website (was: Re: Odp: Jack1 && jack_get_uuid_for_client_name)</title>
<style>
pre {background: #fafafa; border: 1px solid #f1f2f3; font-size: 1.2em; padding: 10px; white-space: pre-wrap; white-space: -moz-pre-wrap; white-space: -pre-wrap; white-space: -o-pre-wrap; word-wrap: break-word;}
h1 {font-size: 140%;}
.header_table {table-layout: fixed; width: 100%;}
.col1 {width: 120px; vertical-align: top;}
.from {font-weight: bold;}
</style>
</head>
<body>
<h1>[Jack-Devel] Restoring trac website (was: Re: Odp: Jack1 && jack_get_uuid_for_client_name)</h1>
<table class="header_table">
<tr><td class='col1'><a href='../1358459875.28958_0.ltw:2,a/index.html'>Prev</a></td><td><a href='../1358523281.22148_0.ltw:2,a/index.html'>Next</a>&nbsp;&nbsp;<a href='../../index.html#1358460810.30542_0.ltw:2,a'>Index</a></td></tr>
<tr><td class='col1'>Date</td><td>Fri, 18 Jan 2013 08:43:14 +1030 </td></tr>
<tr><td class='col1'>From</td><td><span class="from"> Jonathan Woithe </span> &lt;[hidden] at just42 dot net&gt; </td></tr>
<tr><td class='col1'>To</td><td>Paul Davis &lt;[hidden] at linuxaudiosystems dot com&gt; </td></tr>
<tr><td class='col1'>Cc</td><td>jack-devel &lt;[hidden] at lists dot jackaudio dot org&gt; </td></tr>
<tr><td class='col1'>In-Reply-To</td><td>Paul Davis
<a href='../1358426572.10961_0.ltw:2,a/index.html'>Re: [Jack-Devel] Odp: Jack1 &amp;&amp; jack_get_uuid_for_client_name </a></td></tr>
</table>
<pre>
Hi Paul

On Thu, Jan 17, 2013 at 07:42:44AM -0500, Paul Davis wrote:
&gt; &gt; OFC http://trac.jackaudio.org/ is broken, so even can't report bug in
&gt; &gt; this way.
&gt; 
&gt; this has been known about for a long time. the people who set up trac have
&gt; not yet fixed it. it was broken by the webhosting service.

Since both this and the FFADO website were set up at about the same time,
the problem with trac.jackaudio.org is probably the same as we had with
subversion.ffado.org: the python mysql module needs to be recompiled.  The
python error trace from trac.jackaudio.org is almost the same as we were
getting from the ffado site so there's a good chance that it has the same
root cause.  If this is indeed the problem and the mysql python source tree
remains on the host (it was in ffado's case) then the exercise should be
almost trivial.

I recall hearing that there was talk of migrating the content elsewhere at
some point, but if this is still some way off and we know who has access to
the existing host it should be possible to restore trac.jackaudio.org with a
minimum of fuss.

Regards
  jonathan
</pre>
<table class="header_table">
<tr><td class='col1'><a href='../1358459875.28958_0.ltw:2,a/index.html'>Prev</a></td><td><a href='../1358523281.22148_0.ltw:2,a/index.html'>Next</a>&nbsp;&nbsp;<a href='../../index.html#1358460810.30542_0.ltw:2,a'>Index</a></td></tr>
</table>
<p><small>1358460810.30542_0.ltw:2,a&nbsp;&lt;20130117221314.GA23022 at marvin dot atrad dot com dot au&gt;</small></p>
<!-- Created with Bash Archive Mail -->
</body>
</html>
